packages/components/loading/src/loading-provider.tsx

```diff
@@ -199,6 +197,9 @@ export const LoadingProvider: FC<LoadingProviderProps> = ({
   )
 }
 
+LoadingProvider.displayName = "LoadingProvider"
+LoadingProvider.__ui__ = "LoadingProvider"
+
```

Comment:
There’s no need to specify `displayName` for components that don't use forwardRef.

Reply:
@kurakke I'm sorry I'm late. 
I have done! Please review!